core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -178,7 +148,6 @@ class TopicDeletionManager(controller: KafkaController,
           .format(replicasThatFailedToDelete.mkString(","), topics))
         controller.replicaStateMachine.handleStateChanges(replicasThatFailedToDelete, ReplicaDeletionIneligible)
         markTopicIneligibleForDeletion(topics)
-        resumeTopicDeletionThread()
```

Comment:
The comment before markTopicIneligibleForDeletion() needs to be adjusted since we don't need to call that method due to preferred leader election.